api/v1alpha1/stage_types.go

```diff
@@ -685,13 +685,15 @@ type StageList struct {
 	Items           []Stage `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
 
-type PromotionInfo struct {
+type PromotionReference struct {
 	// Name is the name of the Promotion
 	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
 	// Freight is the freight being promoted
 	Freight FreightReference `json:"freight" protobuf:"bytes,2,opt,name=freight"`
 	// Status is the (optional) status of the promotion
 	Status *PromotionStatus `json:"status,omitempty" protobuf:"bytes,3,opt,name=status"`
+	// CompletionTime is the time at which the Promotion was completed.
+	CompletionTime *metav1.Time `json:"completionTime,omitempty" protobuf:"bytes,4,opt,name=completionTime"`
```

Comment:
```suggestion
	FinishedAt *metav1.Time `json:"finishedAt,omitempty" protobuf:"bytes,4,opt,name=finishedAt"`
```

maybe? To make the `CreatedAt`, etc. from upstream (and align with e.g. Argo CD).